static/js/theme-map/Maps/Providers/Google.js

```diff
@@ -6,6 +6,7 @@ import { MapProviderOptions } from '../MapProvider.js';
 import { ProviderMap } from '../ProviderMap.js';
 import { ProviderPin } from '../ProviderPin.js';
 import { debounce } from '../../Util/helpers';
+import isRTL from '../../../../webpack/rtl';
```

Comment:
hmm ok this probably shouldn't go in webpack if we want to use it in static js. Maybe this can go in a static/common folder, or do you have any other ideas?

Reply:
mm I think putting it in static/common is good. will update!